aws_lambda_powertools/event_handler/openapi/models.py

```diff
@@ -380,7 +380,7 @@ class Config:
 
 # https://swagger.io/specification/#operation-object
 class Operation(BaseModel):
-    tags: Optional[List[Tag]] = None
+    tags: Optional[List[str]] = None
```

Comment:
To be triple sure -- that's not a breaking change, right?

I didn't see in the docs an example where they could pass an Operation obj along with a Tag obj

Reply:
It can be considered a breaking change since we're changing the type, but it's also a bug as the original implementation is 100% incorrect and doesn't match the spec.

Reply:
> 100% incorrect and doesn't match the spec

Bug then.